wifiphisher/pywifiphisher.py

```diff
@@ -27,6 +27,7 @@
 import wifiphisher.common.accesspoint as accesspoint
 import wifiphisher.common.tui as tui
 import wifiphisher.common.opmode as opmode
+import wifiphisher.common.victim as victim
```

Comment:
It looks like it's not necessary to import this here.

Reply:
we use it to initiate Victims() dictionary which is a singleton